plugins/modules/mso_schema_template_service_graph.py

```diff
@@ -0,0 +1,264 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: mso_schema_template_service_graph
+short_description: Manage service graph in schema templates
+description:
+- Manage service graphs in schema templates on Cisco ACI Multi-Site.
+author:
+- Shreyas Srish (@shrsr)
+options:
+  schema:
+    description:
+    - The name of the schema.
+    type: str
+    required: yes
+  template:
+    description:
+    - The name of the template.
+    type: str
+    required: yes
+  service_graph_name:
+    description:
+    - The name of the service graph to manage.
+    type: str
+    required: yes
+  service_graph_description:
+    description:
+    - The description of service graph.
+    type: str
+    default: ''
+  service_graph_display_name:
+    description:
+    - The name as displayed on the MSO web interface.
+    type: str
+  service_node_list:
+    description:
+    - A list of nodes to be associated with the service graph.
+    type: list
+    suboptions:
+      type:
+        description:
+        - The type of node
+        required: true
+        type: str
+        aliases: [ name ]
+  node_filter:
+   description:
+    - The filter for the node.
```

Comment:
The filter applied after the first node.

Reply:
ok